ParcelDistributionCenter/ParcelDistributionCenter.Model/json/packages.json

```diff
@@ -274,7 +274,7 @@
   {
     "package_number": 8028221,
     "status": "stored_by_sender",
-    "courier_id": "63984b2636870240c999022e",
+    "courier_id": "63984b26cbf6ee862293e39d",
```

Comment:
A te _id się powtarzały jakoś?

Reply:
Zmieniłem chociaż jedno courier_id żeby mieć przypisane 2 paczki do jednego kuriera i zobaczyć jak to się wyświetla.